addons/auth_oauth/views/res_users_views.xml

```diff
@@ -10,7 +10,7 @@
                         <group>
                             <field name="oauth_provider_id"/>
                             <field name="oauth_uid"/>
-                            <field name="oauth_access_token"/>
+                            <button name="remove_oauth_access_token" type="object" string="Remove Token" invisible="not has_oauth_access_token"/>
```

Comment:
```suggestion
                            <button name="remove_oauth_access_token" type="object" string="Remove Token" invisible="not has_oauth_access_token" groups="base.group_erp_manager"/>
```
?

In any case this view is not the view visible by the user when checking its profile. it's the view in the settings / admin.